file_tree_generator.py

```diff
@@ -0,0 +1,144 @@
+import os
+import regex as re
+
+directories = ["backend", "frontend"]
+
+# Content to ignore
+ignored_directories = [
+    ".next",
+    "node_modules",
+    ".venv",
+    "__pycache__",
+    "aws_rekognition_utils",
+    "aws_secrets_utils",
+    "lambda_utils",
+    "s3_utils",
+    "sqs_utils",
+    "dlp_logging",
+    "endpoints",
+    "firebase_helpers",
+    "tests",
+    "tmp",
+]
+ignored_files = []

```

Comment:
purpose of this variable?

Reply:
The purpose of this variable is to allow extensibility. If, at some point in the future, there ever exists some files that need to be excluded from the directory tree, simply add a file name or a regex pattern to exclude it.